src/prop/sat_proof_manager.h

```diff
@@ -414,8 +414,11 @@ class SatProofManager : protected EnvObj
    *
    * @param conclusion the node-level conclusion of the resolution chain
    * @param conclusionLits the set of literals in the conclusion
+   * @param clauseLevel the level of the clause in the SAT solver (if any)
    */
-  void endResChain(Node conclusion, const std::set<SatLiteral>& conclusionLits);
+  void endResChain(Node conclusion,
+                   const std::set<SatLiteral>& conclusionLits,
+                   uint32_t clauseLevel = -1);
```

Comment:
Should this be uint with a negative default value?

Reply:
It's -1 because it'll become the max which is guaranteed to not trigger the behavior of tracking the clause. I'll clarify in the documentation.